_i18n/en/keyboards/vitamins_included.md

```diff
@@ -39,7 +39,9 @@ Rev2.0 is missing the pull-down resistors on the CCx pins. This means that it wo
 ## Source
 The case was designed in Fusion 360. The right half is just a mirrored left-half
 
-* [Left flat-case source](https://a360.co/2IwRsjE)
+* [Left flat-case source](https://a360.co/2yVzpj2)
+* [Wedge source](https://a360.co/3fIED2e)
+* [Plate source](https://a360.co/2TqNtYX)
```

Comment:
Plate source link seems to have an empty model, but other than that LGTM.

Reply:
Right, the plate is just a dxf, wasn't aware that a360 kinda choked on that. I'll just extrude it to 1.5mm :)

Reply:
Thanks for the review though. I had linked the wrong file for the left case. I had linked the left PCB model instead of the case with that in it :D